Assets/HoloToolkit/SpatialUnderstanding/Scripts/SpatialUnderstanding.cs

```diff
@@ -47,6 +50,12 @@ public bool AllowSpatialUnderstanding
                 return true;
             }
         }
+
+        /// <summary>
+        /// Has any mesh sectors been scanned
+        /// </summary>
+        public bool HasMeshSectors { get { return UnderstandingCustomMesh.HasMeshSectors; } }
```

Comment:
if `UnderstandingCustomMesh.HasMeshSectors` is public why even add this property?